.cron.yml

```diff
@@ -30,8 +30,4 @@ jobs:
           type: decision-task
           treeherder-symbol: legacy-api-ui
           target-tasks-method: legacy_api_ui_tests
-      when:
-          - {hour: 9, minute: 0}
-          - {hour: 12, minute: 0}
-          - {hour: 18, minute: 0}
-          - {hour: 22, minute: 0}
+      when: [] # unscheduled
```

Comment:
Agreed to remove this section in a followup. We'll have to remove these lines too: 

https://github.com/mozilla-mobile/firefox-android/blob/76c939933db01810d002c58c0c3dcb6297c5b933/taskcluster/android_taskgraph/target_tasks.py#L163-L170